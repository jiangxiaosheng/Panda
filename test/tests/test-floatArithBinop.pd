func main() {
	var f1 = 1.0
    var f2 = 2.0

    f1 + f2
    f1 - f2
    f1 * f2
    f1 / f2
}