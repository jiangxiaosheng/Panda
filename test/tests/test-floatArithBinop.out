Sementically checked program: 


void main() {
var f1: float(float : 1.)
var f2: float(float : 2.)
(float : (float : f1) + (float : f2))
(float : (float : f1) - (float : f2))
(float : (float : f1) * (float : f2))
(float : (float : f1) / (float : f2))
}